project/First-Project/pom.xml

```diff
@@ -84,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.modelmapper</groupId>
```

Comment:
Where this is used ?

Reply:
Deleted it now, it was for something I don't use anymore.